src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -4090,13 +4090,13 @@ func main() {
     // TODO: Handle error.
   }
 
-  // Project ID for this request.
+  // * Project ID for this request.
```

Comment:
In C# we have a general TODO above all the placeholders, and no TODOs for each parameter. Here, we do the opposite. Can we be consistent across languages? (AFAIK, there are no language-specific reasons to have diverging styles on this point, but correct me if I'm wrong)